build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
     id "com.github.johnrengelman.shadow" version "5.2.0"
 }
 
-mainClassName = 'echoserver.Main'
+mainClassName = 'Main'
```

Comment:
Are we still going to serve the echo server as well as the HTTP server? I would expect to see another main for the new server.

Reply:
I thought we were allowed only one mainClassName.  Essentially gradle run invokes the Main class not the echoserver main class.